unit-test/deleteOne.js

```diff
@@ -0,0 +1,29 @@
+// import {storageData} from './storage.js';
+// const getTask =  storageData.map((task) => task);
+// getTask[1].completed =true
+// export const deleteTask = () => {
+//   const clearTask = document.querySelector('.clear-task');
+//   clearTask.addEventListener('click', () => {
+//     if (getTask.length > 0) {
+//       const filterCompliteTask = getTask.filter(
+//         (task) => task.completed !== true,
+//       );
+//       localStorage.setItem('Task-list', JSON.stringify(filterCompliteTask));
+//       return filterCompliteTask
+//     }
+//   });
+// };
```

Comment:
- [ ] Please do not leave unused code as a comment in your codebase. this will make your code much noisy, It is always a good practice to keep your code clean from such comments.